### PR TITLE
Overview screen

### PR DESCRIPTION
**Pull request by sirromdev**
https://www.figma.com/file/L89YEqsCYo0yZ07Lwc5dVBaJ/Multi-Collateral-CDP-Portal?node-id=7223%3A138

### CODE REVIEW

File: src/components/RatioDisplay.js [hunk 1]
```diff
@@ -6,23 +6,25 @@ import { Text } from '@makerdao/ui-components-core';
 const CDP_SAFETY_COLOR_PALETTE = {
   [CDP_SAFETY_LEVELS.DANGER]: getColor('orange.500'),
   [CDP_SAFETY_LEVELS.NEUTRAL]: getColor('grey.300'),
-  [CDP_SAFETY_LEVELS.SAFE]: getColor('teal.500')
+  [CDP_SAFETY_LEVELS.SAFE]: getColor('teal.500'),
+  neutral_inverse: getColor('steel')
```
Comment (sirromdev) [inline src/components/RatioDisplay.js]: `grey.300` was too light against the white background needed a darker variant

File: src/components/RatioDisplay.js [hunk 2]
```diff
@@ -6,23 +6,25 @@ import { Text } from '@makerdao/ui-components-core';
...
-export default function RatioDisplay({ ratio, active }) {
+export default function RatioDisplay({ ratio, active, inverse, t }) {
   if (!ratio) return null;
-  const safetyLevel = lookupCDPSafetyLevel(ratio);
+  const safetyLevel = lookupCDPSafetyLevel(ratio, inverse);
 
   return (
     <Text
-      t="p6"
+      t={t ? t : 'p6'}
```
Comment (sirromdev) [inline src/components/RatioDisplay.js]: The table component text defaults to "p3" and this is a workaround to fix that. Maybe it's bit hacky but let me know what you think

**Comment by levity** [comment]
@michaelelliot I’m tagging you on this one since your work week has already started so you can help move it along :)

**Comment by b-pmcg** [comment]
Got it up and running locally after the latest changes, looks good.

**Pull request merged**